react-native-libraries.json

```
@@ -10485,5 +10485,25 @@
     "examples": ["https://github.com/Malaa-tech/expo-squircle-view/tree/main/example"],
     "ios": true,
     "android": true
+  },
+  {
+    "githubUrl": "https://github.com/AnkiPro/react-native-canvas-view",
+    "npmPkg": "@ankipro/react-native-canvas-view",
+    "examples": ["https://github.com/AnkiPro/react-native-canvas-view/tree/main/example"],
+    "images": ["https://github.com/AnkiPro/react-native-canvas-view/blob/main/assets/demo.png"],
+    "ios": true
+  },
+  {
+    "githubUrl": "https://github.com/AnkiPro/react-native-file-open",
+    "npmPkg": "@ankipro/react-native-file-open",
+    "examples": ["https://github.com/AnkiPro/react-native-file-open/tree/main/example"],
+    "ios": true
+  },
+  {
+    "githubUrl": "https://github.com/AnkiPro/react-native-window-resize",
+    "npmPkg": "@ankipro/react-native-window-resize",
+    "examples": ["https://github.com/AnkiPro/react-native-window-resize/tree/main/example"],
+    "images": ["https://github.com/AnkiPro/react-native-window-resize/raw/main/assets/example.gif"],
+    "ios": true
```

Hello @tokkozhin, thank you for adding new libraries to the directory! 👍 

```suggestion
    "ios": true,
    "dev": true
```

Got only one suggestion, since `@ankipro/react-native-window-resize` looks like a package mostly tailored to improve development process, let's add the [`dev` tag](https://github.com/react-native-community/directory?tab=readme-ov-file#dev) to it. Besides that nit, everything looks great!

---

Yes, agree with you. Thx for suggestion!